formats/capabilities/SPEC_1.0.md

```diff
@@ -0,0 +1,91 @@
+# Capability format
+
+<dl>
+	<dt>Namespace</dt><dd>`capabilities`</dd>
+	<dt>Version</dt><dd>1.0</dd>
+</dl>
+
+This is an application format for continuous sharing and minting of capability information
+
+## Subspace
+
+Capabilities are by an entity, and are therefore published under the issuing entities subspace.
+
+## Public capabilities
+
+The knowledge that you've issued capabilities to the receiver will be public for all
```

Comment:
Probably won't hurt to mention here that the *contents* are not public.

Reply:
Fair, have added a small note on that :)